client/verta/tests/constants.py

```diff
@@ -29,7 +29,6 @@
 DEV_KEY_2 = os.environ.get(DEV_KEY_2_ENV_VAR)
 EMAIL_3 = os.environ.get(EMAIL_3_ENV_VAR)
 DEV_KEY_3 = os.environ.get(DEV_KEY_3_ENV_VAR)
-WORKSPACE = os.environ.get(WORKSPACE_ENV_VAR, DEFAULT_ENV)
```

Comment:
Do you think we should remove `WORKSPACE_ENV_VAR` and `DEFAULT_ENV` from this file, too?

Reply:
I think it is not used and was introduced in VUMM-760_remove_organization PR. But can revert it if we plan to use it somehow in the future. Reverted